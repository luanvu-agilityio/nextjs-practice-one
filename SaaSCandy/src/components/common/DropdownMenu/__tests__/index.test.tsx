import { render } from '@testing-library/react';
import {
  DropdownMenu,
  DropdownMenuTrigger,
  DropdownMenuContent,
  DropdownMenuItem,
  DropdownMenuLabel,
  DropdownMenuSeparator,
} from '../index';

describe('DropdownMenu', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <DropdownMenu>
        <DropdownMenuTrigger>Open</DropdownMenuTrigger>
        <DropdownMenuContent>
          <DropdownMenuLabel>My Account</DropdownMenuLabel>
          <DropdownMenuSeparator />
          <DropdownMenuItem>Profile</DropdownMenuItem>
          <DropdownMenuItem>Settings</DropdownMenuItem>
        </DropdownMenuContent>
      </DropdownMenu>
    );
    expect(container).toMatchSnapshot();
  });
});
