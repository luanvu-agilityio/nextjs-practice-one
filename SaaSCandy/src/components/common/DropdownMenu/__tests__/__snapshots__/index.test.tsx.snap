// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`DropdownMenu matches snapshot 1`] = `
<div>
  <button
    aria-expanded="false"
    aria-haspopup="menu"
    data-state="closed"
    id="radix-«r0»"
    type="button"
  >
    Open
  </button>
</div>
`;
